### PR TITLE
Chalk 1.x definitions

### PR DESCRIPTION
**Pull request by thejameskyle**
This was adapted from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/chalk/chalk.d.ts

### CODE REVIEW

File: definitions/npm/chalk_v1.x.x/flow_>=v0.21.x/chalk_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+type ChalkStyleElement = {
```
Comment (ryyppy) [inline definitions/npm/chalk_v1.x.x/flow_>=v0.21.x/chalk_v1.x.x.js]: Also here, just namespace with `$npm$chalk$...` :-)

**Comment by ryyppy** [comment]
Awesome to see PRs from a facebook newcomer 👍  🎉

**Comment by thejameskyle** [comment]
Updated

**Comment by marudor** [comment]
Looks nice for me - the failed tests are due to the .28 release - already on it.
Nothing keeps us from merging this.

**Pull request merged**